### PR TITLE
ENH skip ci services during token registration

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Added a ``news`` entry

### CODE REVIEW

File: conda_smithy/feedstock_tokens.py [hunk 1]
```diff
@@ -302,7 +302,7 @@ def register_feedstock_token(user, project, token_repo):
 
             # push
             repo.index.add(token_file)
-            repo.index.commit("added token for %s/%s" % (user, project))
+            repo.index.commit("[ci skip] [skip ci] [cf admin skip] added token for %s/%s" % (user, project))
```
Comment (isuruf) [inline conda_smithy/feedstock_tokens.py:305]: Can you add the one for Azure as well?

**Comment by isuruf** [comment]
@beckermr, can you stop the admin migration until this is in? We don't want to be releasing packages without the maintainers knowing.

**Comment by beckermr** [comment]
This is for commits to the feedstock token repo, not to feedstocks

**Comment by beckermr** [comment]
So we are not releasing packages

**Comment by beckermr** [comment]
Also, azure respects `[ci skip]` now, but I can add the `***NO_CI***` one too.

File: conda_smithy/feedstock_tokens.py (hunk 1 above)
Comment (beckermr) [reply conda_smithy/feedstock_tokens.py]: done

**Comment by beckermr** [comment]
Good to merge @isuruf?

**Comment by beckermr** [comment]
thanks!

**Pull request merged**